pkgs/by-name/or/or-tools/scip.patch

```diff
@@ -0,0 +1,145 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8492dc75..4c12a9bf 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -38,9 +38,11 @@ set(CPACK_PACKAGE_VENDOR "Zuse Institute Berlin")
+ set(CPACK_PACKAGE_CONTACT "http://scipopt.org")
+ include(CPack)
+ 
+-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
+-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
+-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
++if(CMAKE_PROJECT_NAME EQUAL "SCIP")
```

Comment:
I don't see this lists included from elsewhere, surprised there would be any other branch...

EDIT: ok looking at targets below I guess it actually is imported...